### PR TITLE
Make ap4k work better with jdk11.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,6 +38,9 @@ hs_err_pid*
 
 *.iml
 
+#
+# idee
+.idee
```
Comment (metacosm) [inline .gitignore]: idee?

File: core/src/main/java/io/ap4k/kubernetes/decorator/AddRoleBindingDecorator.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+package io.ap4k.kubernetes.decorator;
+
+import io.ap4k.Resources;
+import io.ap4k.deps.kubernetes.api.model.KubernetesListBuilder;
+import io.ap4k.deps.kubernetes.api.model.rbac.KubernetesRoleBindingBuilder;
+
+/**
+ * AddRoleBindingDecorator
+ */
+public class AddRoleBindingDecorator extends Decorator<KubernetesListBuilder> {
```
Comment (metacosm) [inline core/src/main/java/io/ap4k/kubernetes/decorator/AddRoleBindingDecorator.java]: How is this relevant to making things work better with Java 11?

File: .gitignore (hunk 1 above)
Comment (iocanel) [reply .gitignore]: It's where my custom IDE is storing project specific configuration.

File: core/src/main/java/io/ap4k/kubernetes/decorator/AddRoleBindingDecorator.java (hunk 2 above)
Comment (iocanel) [reply core/src/main/java/io/ap4k/kubernetes/decorator/AddRoleBindingDecorator.java]: It won't. I guess I need to rebase.

**Comment by iocanel** [comment]
@metacosm: would you like to review it now that I rebased?
